src/pages/reference/accounts.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Accounts
+---
+
+Railway Accounts are how a user interacts with the Railway platform. 
+
+Accounts currently are only allowed to be linked to one email and one GitHub account.
+
+Users are only allowed one account per person, if caught multi-accounting, we do not hesitate to ban.
+
+## Account Actions
+
+The main account preferences page is accessible by clicking your profile photo in the top right and selecting `Account Settings` 
+
+### Change Account Information
+
+Accounts can change their display name, profile photo, and account email under the main account 
+
+## Account Security
+
+Railway is committed to you and your project's security. As such, we provide a variety of methods to help you keep your piece of mind.
+### Enabling 2FA 
+
+You can enable Two-Factor Authentication under the Account Security page. Most TOTP applications are supported.
+
+After scanning the provided QR code and entering the code for the initial pairing of the application, 2FA will require additional verification for login and destructive account actions. 
+
+### Account Sessions
+
+You can now see all the places you are logged in on the [account security page](https://railway.app/account/security). This includes all browser and CLI sessions that have logged in since today. Revoking a session will immediately log that device out.
```

Comment:
```suggestion
You can see all the places you are logged in on the [account security page](https://railway.app/account/security). This includes all browser and CLI sessions that have logged in since today. Revoking a session will immediately log that device out.
```